scipy/optimize/_linprog_rs.py

```diff
@@ -342,9 +342,9 @@ def _phase_two(c, A, x, b, callback, postsolve_args, maxiter, tol, disp,
             else:
                 phase = 2
                 x_postsolve = x
-            x_o, fun, slack, con, _ = _postsolve(x_postsolve,
-                                                    postsolve_args,
-                                                    tol=tol, copy=True)
+            x_o, fun, slack, con = _postsolve(x_postsolve,
+                                              postsolve_args,
+                                              copy=True)
```

Comment:
Looks like `copy` can be removed, too.

Reply:
It certainly looks that way. Do you remember why it was in there?

Reply:
Yes, now I do. When the option `disp=True` or when using the callback, `_postsolve` gets called every iteration. When originally written, `_postsolve` was only supposed to be run "post-solve", not every iteration, so it added variables back into the solution vector `x` in-place. Interior point didn't care, because it actually passes in `x/tau`, and legacy simplex effectively copies with some fancy indexing before passing `x` in, but `revised simplex` didn't copy, so we (I?) added the argument, and `_postsolve` performed the copy only if necessary - only if variables would be added back. But now I think `_unscale` (which was added later) guarantees that a copy is made, so I think it's safe to remove.

Reply:
I removed it.